### PR TITLE
Added roadmap update Q1 2022

### PR DESCRIPTION
**Pull request by sujaya-sys**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -2,41 +2,23 @@
...
 
-## Q3 2021
+### Improved Resource Driver Management
 
-### Deployment Pipelines
+Resource Drivers are used to establish a connection between Humanitec and your infrastructure. There is a growing number of drivers available out-of-the-box in Humanitec. In this quarter we aim to improve their handling and flexibility. This includes the provisioning of generic drivers as well as the open-sourcing of drivers to simplify usage, extention and adapation for specific needs. Additionally are we planning to introduce virtual drivers. Since drivers can be implemented using the technology of other drivers, a virtual driver allows to do so by simply forwarding its inputs to another driver. A comprehensive overview of the Humanitec driver ecosystem as well as its integration and contribution options will be published in a central hub on our Website.
```
Comment (chrishumanitec) [inline docs/roadmap.md:17]: I suggest some changes:
> Resource Drivers are used to provision things that are consumed by workloads. Most commonly this is infrastructure such as databases or dns. There are a growing number of drivers available out-of-the-box in Humanitec. In this quarter we aim to improve the flexibility and range of our driver offering. This includes adding generic drivers that work with templates and IaC and open-sourcing many of our drivers. We also plan to extend how drivers can be defined - allowing drivers to be defined in terms of other drivers. This should make it easier for ops teams to define good defaults to reuse within their teams. A comprehensive overview of the Humanitec driver ecosystem as well as its integration and contribution options will be published in a central hub on our Website.

File: docs/roadmap.md [hunk 2]
```diff
@@ -2,41 +2,23 @@
...
 
-### UI update: fresh looks and a more intuitive interface
+### App Details Page UI Update
 
 Following several months of user testing and customer feedback we are planning on introducing a number of changes to improve the usability, predictability and scalability of our UI. The focus lies on the App Details Page, the control centre of an App. We want to address issues related to interaction design, information hierarchy, status communication as well as aesthetics while touching on UI patterns which are likely to change the look and feel of the Humanitec UI as a whole.
 
-## Q3 2021
+### Improved Resource Driver Management
```
Comment (chrishumanitec) [inline docs/roadmap.md:15]: How about:
> Increase Resource Driver range and flexibility

**Review by chrishumanitec — CHANGES_REQUESTED** [review]
Looks good. I suggested some changes to the  Resource Driver part.

**Review by christophcrichter — APPROVED** [review]
LGTM

**Review by christophcrichter — APPROVED** [review]
LGTM

**Pull request merged**